geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientUpdater.java

```diff
@@ -1598,7 +1595,7 @@ private void processMessages() {
 
         try {
           // Read the message
-          clientMessage.recv();
+          clientMessage.receiveWithHeaderReadTimeout(serverQueueStatus.getPingInterval() * 2);
```

Comment:
Shouldn't we be multiplying by the subscription timeout multiplier here instead of two?

Reply:
ooh - good catch Brian